usaspending_api/database_scripts/matview_generator/mv_loan_award_search.json

```diff
@@ -110,6 +111,16 @@
     "    ON (awards.latest_transaction_id = transaction_fabs.transaction_id AND latest_transaction.is_fpds = false)",
     "LEFT OUTER JOIN",
     "  (SELECT",
+    "    award_id, jsonb_agg(DISTINCT jsonb_build_object('cfda_number', cfda_number, 'cfda_program_title', cfda_title))::text AS cfdas",
```

Comment:
Should we need to read this value and parse it in the API might be better to store as: ```array_agg(DISTINCT jsonb_build_object('cfda_number', cfda_number, 'cfda_program_title', cfda_title)::text) ```

and updating Contract and IDV to `NULL::text[] AS cfdas`. 

Main reason is that Elasticsearch will store an Array of Objects, Text, etc. and query against all of them when looking for a match.